Automattic/vip-go-mu-plugins
# Add/test jquery updates

## Description

Adds [Test JQuery Updates ](https://wordpress.org/plugins/wp-jquery-update-test/) plugin into `mu-plugins`. The plugin is enabled if `VIP_ENABLE_TEST_JQUERY_UPDATES` constant is set.

Note that the current version is meant for WP >5.6.X, so it won't show anything on lower versions.

To get 5.5.x version you can run to get back to that revision (note that this will create git commits in your repo so please revert those afterwards):
`./bin/update-test-jquery-updates.sh b9e54c55bbadcb21fb606fa185146f314262b3c8` 
 

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. checkout the pr
1. run `./bin/update-test-jquery-updates.sh b9e54c55bbadcb21fb606fa185146f314262b3c8`
1. See the new admin menu under `plugins`
